app/test_main.py

```diff
@@ -1,3 +1,37 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_first_cat_and_dog_years():
+    assert (
+        get_human_age(14, 14) == [0, 0]
+    )
+
+
+def test_first_cat_and_dog_years_with_a_critical_value():
+    assert (
+        get_human_age(15, 15) == [1, 1]
+    )
+
+
+def test_second_cat_and_dog_years():
+    assert (
+        get_human_age(23, 23) == [1, 1]
+    )
+
+
+def test_third_cat_and_dog_years():
+    assert (
+        get_human_age(27, 27) == [2, 2]
+    )
+
+
+def test_second_cat_and_dog_years_with_a_difference_age():
+    assert (
+        get_human_age(28, 28) == [3, 2]
+    )
+
+
+def test_third_years_with_a_critical_value():
+    assert (
+        get_human_age(100, 100) == [21, 17]
+    )
```

Comment:
Consider using `@pytest.mark.parametrize` to avoid repeating similar test cases. This will make your tests cleaner and easier to maintain, as suggested in the checklist.